src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -26,6 +26,7 @@
 @SuppressWarnings("WeakerAccess")
 public class Main extends GuiApplication {
 
+  // TODO - sync these values with build.gradle
```

Comment:
Any ideas on how we can create a common spot to access these vars for both `Main.java` and `build.gradle`?

I don't think it's a deal-breaker if we can't, as I typically just Ctrl-Shift-F in IntelliJ and search for the current version number any time I want to update it (which would catch both spots). Probably safe to get rid of both TODOs in that case, and maybe create an issue for the backlog if you think it's worth it.

Reply:
I think having a standalone `VERSION` or `version.txt` would be sufficient, but I need to read more about best practices in Java/Gradle to make that call.

My plan was to (1) create a ticket, (2) add the ticket number to the TODO. Whenever I need to copy/paste changes across a codebase, I like to also copy/paste a comment nearby that tells me all the places it needs to be synced.

Reply:
That sounds perfect.

Reply:
https://github.com/BrightSpots/rcv/issues/662 and also noted in the comments now, feel free to add any context needed to that issue